src/Workspaces/Core/Portable/Workspace/Solution/Document.cs

```diff
@@ -200,6 +202,8 @@ public Task<SyntaxTree> GetSyntaxTreeAsync(CancellationToken cancellationToken =
 
         /// <summary>
         /// Gets the root node of the current syntax tree if it is available.
```

Comment:
To me I'm reading it as meeting the previous criteria of "if the text is already loaded and the tree is already parsed."

But seriously, if it's ambiguous to you, just send a new PR with something that makes more sense to you. I don't say that to be flippant, but I'm not psychic and can't fix problems I don't see.